src/posix/platform/CMakeLists.txt

```
@@ -53,6 +53,17 @@ if(NOT OT_CONFIG)
     set(OT_CONFIG ${OT_CONFIG} PARENT_SCOPE)
 endif()
 
+option(OT_RESET_CONNECTION "reset connection after hard resetting RCP(USB CDC ACM) device" OFF)
```

```suggestion
option(OT_SPINEL_RESET_CONNECTION "reset connection after hard resetting RCP(USB CDC ACM) device" OFF)
```

`OT_RESET_CONNECTION` seems too general.